src/lib/modules/Document/DocumentToc.js

```diff
@@ -2,14 +2,14 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import Overridable from 'react-overridable';
 import { List } from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
We try to keep the `_` to remember that it is `lodash`
```suggestion
import _isEmpty from 'lodash/isEmpty';
```